### PR TITLE
depricate report-dir and reportDir usage

### PR DESCRIPTION
**Pull request by pinikomarov**
not needed since : https://github.com/onsi/ginkgo/pull/601

### CODE REVIEW

File: test/e2e/framework/test_context.go [hunk 1]
```diff
@@ -47,7 +46,6 @@ func init() {
 		"Path to kubeconfig containing embedded authinfo.")
 	flag.Var(&TestContext.KubeContexts, "dp-context", "kubeconfig context for dataplane clusters (use several times).")
 	flag.StringVar(&TestContext.ReportPrefix, "report-prefix", "", "Optional prefix for JUnit XML reports. Default is empty, which doesn't prepend anything to the default name.")
```
Comment (manosnoam) [inline test/e2e/framework/test_context.go:48]: How would it effect "report-prefix" option ?

**Comment by tpantelis** [comment]
Is this PR still relevant?

**Comment by manosnoam** [comment]
I think it is still relevant.
we should use ginkgo option in Submariner tests run: `-ginkgo.reportFile
<file path>`

**Comment by tpantelis** [comment]
This PR has been open for quite a while - if it's still relevant, can we make progress on it? Otherwise let's close it.

**Comment by manosnoam** [comment]
@pinikomarov  I'm closing this for now, please reopen when you find the time, thanks!

**Pull request closed without merging**